### PR TITLE
fix(request): skip requests if a parameter is set to 'nil'

### PR DESCRIPTION
**Pull request by MachadoLhes**
It should only skip a request if the query parameter is not present
Solving issue B2W-BIT/restQL-http/issues/36

### CODE REVIEW

File: test/integration/restql/restql_test.clj [hunk 1]
```diff
@@ -248,6 +248,14 @@
     (let [result (execute-query uri "from hero with name = $name" {:name "Dwayne \"The Rock\" Johnson"})]
       (is (= 200 (get-in result [:hero :details :status]))))))
 
+(deftest request-with-missing-param
+  (with-routes!
+    {{:path "/hero" :query-params {:name "Dwayne+%22The+Rock%22+Johnson"}} (hero-route)}
+    (let [result (execute-query uri "from hero with name = $name, id = $id" {:name "Dwayne \"The Rock\" Johnson"})
+          result2 (execute-query uri "from hero with name = $name, id = $id" {:name "Dwayne \"The Rock\" Johnson" :id nil})]
```
Comment (cupello) [inline test/integration/restql/restql_test.clj]:
```
(let [query  "from hero with name = $name, id = $id"
       result_without_param (execute-query uri query {:name "Dwayne \"The Rock\" Johnson"})
       result_with_param (execute-query uri query {:name "Dwayne \"The Rock\" Johnson" :id nil})]
```

Comment (lucasbarros) [reply test/integration/restql/restql_test.clj]: For me this seems more correct if it is divided into two test cases

```clojure
(testing "Without param"..
(testing "With Param"..
```

**Comment by lucasbarros** [comment]
We are skipping requests in some cases of chained calls because the param is unavailable.
I think we should solve the optional parameter (B2W-BIT/restQL-http/issues/84) before approving this issue.

**Comment by cupello** [comment]
I am closing this PR because some requirements are missing.

**Pull request closed without merging**